spec/lib/goodcity/organisation_populator_spec.rb

```diff
@@ -29,22 +29,23 @@
   end
 
   context "update organisation" do
+    let(:registration_id_one) { "91/09657"}
```

Comment:
@trojanh Use `let` syntax instead of `@instance` variables. `let` has delayed evaluation so it's not run until it's called.